src/main/java/roboy/util/IO.java

```diff
@@ -43,4 +51,60 @@ public static String readFile(File file){
 			return new ArrayList<>();
 		}
 	}
+

```

Comment:
Not related to new code but GitHub won't let me comment the old code:
- All the methods above (readFile and readLines) are not used 
-  `new String(Files.readAllBytes ...` might break for some character encodings since no encoding is specified
- I would remove readFile() methods to prevent possible errors

Reply:
created an issue for this: https://devanthro.atlassian.net/browse/R20D-183